content/uk/docs/reference/glossary/userns.md

```diff
@@ -0,0 +1,23 @@
+---
+title: user namespace
```

Comment:
We could localize this too, with `userns` as the AKA.

Reply:
Done. See suggestion.